package-lock.json

```diff
@@ -446,12 +446,56 @@
           "resolved": "https://registry.npmjs.org/babel-register/-/babel-register-6.26.0.tgz",
```

Comment:
Don't commit this file, this is 446 LOC which has absolutely nothing to do with the PR